test/account.py

```diff
@@ -155,7 +155,7 @@ def get_account_config_contents(
         }
     }
     if verify is not None:
-        out["verify"] = verify
+        out[name]["verify"] = verify
     if proxies is not None:
-        out["proxies"] = proxies
+        out[name]["proxies"] = proxies
```

Comment:
I don't think we ever supported "global" configuration values for the new account management approach. Setting up the configuration file in an invalid way from our test cases lead to test issues.